scipy/stats/_covariance.py

```diff
@@ -4,13 +4,122 @@
 from scipy import linalg
 
 
-__all__ = ["CovViaPrecision"]
+__all__ = ["Covariance"]
 
 
 class Covariance:
     """
     Representation of a covariance matrix
+
+    Calculations involving covariance matrices (e.g. data whitening,
+    multivariate normal function evaluation) are often performed more
+    efficiently using a decomposition of the covariance matrix instead of the
+    covariance metrix itself. This class allows the user to construct an
+    object representing a covariance matrix using any of several
+    decompositions and perform calculations using a common interface.
+
+    Note that the `Covariance` class cannot be instantiated directly.
+    Instead, use one of the factory methods (e.g. `Covariance.from_diagonal`).
+
+    Examples
+    --------
+    The most common use of the `Covariance` class is to call one of the
+    factory methods to create a `Covariance` object, then pass that
```

Comment:
I think we should rephrase the "most common" part as you must now use a factory method.

Reply:
To "The `Covariance` class is used by calling one of the factory methods..."?

I didn't think it was needed because it is not wrong and soon we will be able to instantiate `Covariance` directly, so there will be another way of using it. But if you prefer, I can change it.